### PR TITLE
Adds an OATs setup for EF Core and Sqlite

### PR DESCRIPTION
**Pull request by matt-hensley**
## Changes

Sets up EF Core and Sqlite with minimal OATs testing.

## Merge requirement checklist

* [x] Unit tests added/updated
* [ ] ~[`CHANGELOG.md`](https://github.com/grafana/grafana-opentelemetry-dotnet) file updated for non-trivial changes~
* [ ] ~Changes in public API reviewed (if applicable)~

### CODE REVIEW

File: examples/net6.0/aspnetcore/Controllers/EFCoreController.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
+using Microsoft.AspNetCore.Mvc;
```
Comment (pyohannes) [inline examples/net6.0/aspnetcore/Controllers/EFCoreController.cs:1]: This should also be added to the `net8.0` project.

**Review by pyohannes — COMMENTED** [review]
For some reasons I don't get the EFCore tests to pass, I'll take a closer look.

File: examples/net6.0/aspnetcore/Controllers/EFCoreController.cs (hunk 1 above)
Comment (matt-hensley) [reply examples/net6.0/aspnetcore/Controllers/EFCoreController.cs:1]: This appears in the net8.0 copy of this file and seems to compile correctly locally - am I missing something?

Comment (pyohannes) [reply examples/net6.0/aspnetcore/Controllers/EFCoreController.cs:1]: Changes in the `Program.cs` are missing.

Comment (pyohannes) [reply examples/net6.0/aspnetcore/Controllers/EFCoreController.cs:1]: Changes in the `Program.cs` are missing.

File: docker/docker-compose-aspnetcore/efcore/oats.efcore-blogs.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+include:
+  - ./oats-template.yaml
+input:
+  - path: /api/EFCore/ListBlogs
+expected:
+  traces:
+    - traceql: '{ name =~ "api/EFCore/ListBlogs" }'
+      spans:
+        - name: 'GET'
```
Comment (pyohannes) [inline docker/docker-compose-aspnetcore/efcore/oats.efcore-blogs.yaml:9]:
```suggestion
        - name: 'main'
```

This change is required so tests pass for me.

**Review by pyohannes — COMMENTED** [review]
Two changes I need to do to get tests passing for me.

**Pull request closed without merging**